### PR TITLE
346 fix image in addTagStepImage / addTagStepTrack

### PR DESCRIPTION
**Pull request by JanEberlein**
- image Url of track now gets written into correct field of NFCTag

### CODE REVIEW

**Review by tcoro — APPROVED** [review]
⭕ 🇰🇪

File: packages/app/src/components/add-tag/AddTagStepTrack.vue [hunk 1]
```diff
@@ -109,7 +109,7 @@ export default defineComponent({
 
       const tagCopy = tag.value;
       tagCopy.trackUri = track.uri;
-      tagCopy.imageUrl = track.imageUri;
+      tagCopy.trackImageUrl = track.imageUri;
```
Comment (schnickalodeon) [inline packages/app/src/components/add-tag/AddTagStepTrack.vue:112]: What is the diffrence between `tagCopy.trackImageUrl` and `  tagCopy.imageUrl`
The naming is not quite clear

Is imageUri the external image?

**Review by schnickalodeon — CHANGES_REQUESTED** [review]

File: packages/app/src/components/add-tag/AddTagStepTrack.vue (hunk 1 above)
Comment (JanEberlein) [reply packages/app/src/components/add-tag/AddTagStepTrack.vue:112]: trackImageUrl is the default image from the music service(i. e. spotify). trackImage is the image Url th user chooses

Comment (schnickalodeon) [reply packages/app/src/components/add-tag/AddTagStepTrack.vue:112]: Could we rename that to clarify that?
- externalImageUri
- musicServiceImageUri

or something like this?

Comment (anbraten) [reply packages/app/src/components/add-tag/AddTagStepTrack.vue:112]: @JanEberlein I would simplify that a bit as well. I will work on it and commit my changes.

**Comment by anbraten** [comment]
@schnickalodeon @JanEberlein Could you have a look on my latest changes?

**Review by anbraten — APPROVED** [review]
lgtm

**Comment by JanEberlein** [comment]
> @schnickalodeon @JanEberlein Could you have a look on my latest changes?

Looks good 👍 
Can you approve it? it's my PR 😅

File: packages/app/src/components/add-tag/AddTagStepTrack.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/components/add-tag/AddTagStepTrack.vue:112]: I resolved that ...

**Pull request merged**